arviz/tests/base_tests/test_data_zarr.py

```diff
@@ -26,7 +27,8 @@ def data(self, draws, chains):
         class Data:
             # fake 8-school output
             obj = {}
-            for key, shape in {"mu": [], "tau": [], "eta": [8], "theta": [8]}.items():
+            shapes: Mapping[str, list] = {"mu": [], "tau": [], "eta": [8], "theta": [8]}
```

Comment:
Consider something more specific than `list` so it captures the types within the list? I'm not sure what `np.random.randn` requires the type of `shape` to be, exactly.

Reply:
i was actually surprised this popped up -- this is a literal dict used once, so `List[int]` (or something...) seems like it should have been deduced.